.github/workflows/maven.yml

```diff
@@ -42,7 +42,12 @@ jobs:
           path: ~/.m2/repository
           key: ${{ runner.os }}-maven-${{ matrix.java }}-${{ hashFiles('pom.xml', '**/pom.xml') }}
       - name: Buid with Maven
-        run: mvn -q clean test -B -Dorg.slf4j.simpleLogger.log.org.apache.maven.cli.transfer.Slf4jMavenTransferListener=warn
+        run: |
+          mvn clean test \
+          -q \
+          -Dmaven.wagon.http.ssl.insecure=true \
```

Comment:
I don't think we want to do this, do we? This will enable people to perform man in the middle attacks.